### PR TITLE
When installing using MSI, set the working directory of the shortcut to the user home directory

### LINKED ISSUE #5705: Installer should start PS Core with user's home dir as the working dir
Steps to reproduce
------------------

```powershell
Install PS Core rc2 and at the select check the box to start PowerShell.
Find PowerShell Core in the task bar, right-click it and select "Pin to taskbar"
```

Expected behavior
-----------------
PowerShell Core as started by the installer, starts in my home dir.
PowerShell Core as started by the taskbar icon, starts in my home dir.

Actual behavior
---------------
PowerShell Core in both cases starts in `C:\Program Files\PowerShell\6.0.0-rc.2`.

I wind up having to edit the taskbar shortcut's properties to change the "Start in:" location to `%USERPROFILE%` - which works but I shouldn't have to do this.  I suspect other folks will use the PS Core started by the installer as the easy means to get PS Core pinned to their taskbar.

If the installer could set the working dir to the current user's home dir before launching PowerShell, that would probably fix both scenarios.

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.0-rc.2
PSEdition                      Core
GitCommitId                    v6.0.0-rc.2
OS                             Microsoft Windows 10.0.16299
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
## PR Summary

Leverage the `-WorkingDirectory` parameter on pwsh.exe to set the initial location.

Fix https://github.com/PowerShell/PowerShell/issues/5705

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
Your commit isn't associated with your github id.  Can you fix that?

**Review by bergmeister — APPROVED** [review]
LGTM if you have tested it and it works.

File: assets/Product.wxs [hunk 1]
```diff
@@ -227,7 +227,8 @@
             <Shortcut Id="PowerShell_ProgramsMenuShortcut" 
                       Name="$(var.ProductSimpleVersionWithNameAndOptionalArchitecture)" 
                       Description="$(var.ProductSimpleVersionWithNameAndOptionalArchitecture)" 
-                      Target="[$(var.ProductDirectoryName)]pwsh.exe" 
+                      Target="[$(var.ProductDirectoryName)]pwsh.exe"
+                      Arguments="-WorkingDirectory ~"
                       WorkingDirectory="$(var.ProductDirectoryName)"
```
Comment (iSazonov) [inline assets/Product.wxs]: Seems we can remove the line.

**Comment by TravisEz13** [comment]
I'd suggest addressing @iSazonov 's comment

File: assets/Product.wxs (hunk 1 above)
Comment (SteveL-MSFT) [reply assets/Product.wxs]: Yes, will remove.

**Pull request merged**